pages/cadastro/index.public.js

```diff
@@ -131,7 +132,7 @@ function SignUpForm() {
           )}
         </FormControl>
         <FormControl id="email">
-          <FormControl.Label>Email</FormControl.Label>
+          <FormControl.Label id="signup__label-email">Email</FormControl.Label>
```

Comment:
Dando uma olhada na [documentação do MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#accessibility_concerns), que tal usarmos o atributo `for` na `label` ao invés de `id` e `aria-describedby` para vincular a `label` com o `input`?

No site [a11y-101](https://a11y-101.com/development/aria-describedby) tem um caso de uso melhor para o atributo `aria-describedby`.

Não sou especialista nesses pontos de acessibilidade, então se eu estiver enganado, me avise 👍

Reply:
E aí, @Rafatcb ! Tudo bem? Concordo com o que você disse sobre a troca.
No caso, eu havia atribuído o aria-describedby via ID, pois verifiquei que este atributo está sendo gerado automaticamente com valor vazio. (Fiz essa inspeção via plugin WAVE). Ao invés de deixá-lo vazio, atribuí a ele o valor da label.

Entretanto você trouxe um ponto bem pertinente que, de fato, não é este o objetivo dele. Então em relação a vínculo, farei a troca para o FOR como você sugeriu. Obrigado.